js/chatbot/static/Copy.svelte

```diff
@@ -1,11 +1,10 @@
 <script lang="ts">
 	import { onDestroy } from "svelte";
-	import { fade } from "svelte/transition";
 	import { Copy, Check } from "@gradio/icons";
 
 	let copied = false;
 	export let value: string;
-	let timer: NodeJS.Timeout;
```

Comment:
this was not formally correct as we are dealing with client-side code. IDE protested as well :)

Reply:
Getting a type error in our CI, which is probably due to this change:

```
/home/runner/work/gradio/gradio/js/chatbot/static/Copy.svelte:12:3
Error: Type 'Timeout' is not assignable to type 'number'. (ts)
		if (timer) clearTimeout(timer);
		timer = setTimeout(() => {
			copied = false;
```

Reply:
yes, fixed, thanks